drivers/watchdog/iwdg_stm32.h

```diff
@@ -18,59 +19,22 @@
  *
  * Chapter 19: Independent watchdog (IWDG)
  *
+ *   STM32F37xx advanced ARM ® -based 32-bit MCUs
+ *
+ * Chapter 21: Independent watchdog (IWDG)
```

Comment:
Since this driver targets all STM32 SoCs, I wonder if there would be a better way to put  this reference.